src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -1,13 +1,17 @@
 using System;
 using CoreGraphics;
 using Microsoft.Maui.Graphics;
-using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui.Platform
 {
 	public class LayoutView : MauiView
 	{
+		public LayoutView() 
+		{
+			ClipsToBounds = true;
```

Comment:
Could we include some comments in code in this cases to explain why we are clipping by default?. Just because the behavior is a little different from how it was in Xamarin.Forms.